yoti_python_sdk/tests/test_protobuf.py

```diff
@@ -5,32 +5,27 @@
 
 from yoti_python_sdk.protobuf import protobuf
 
+string_value = "123"
+byte_value = str.encode(string_value)
+int_value = int(string_value)
+
 
 @pytest.fixture(scope='module')
 def proto():
     return protobuf.Protobuf()
 
 
-def test_protobuf_value_based_on_content_type(proto):
-    string_value = "123"
-    byte_value = str.encode(string_value)
-    int_value = int(string_value)
-
-    result = proto.value_based_on_content_type(byte_value, proto.CT_STRING)
-    assert result == string_value
-
-    result = proto.value_based_on_content_type(byte_value, proto.CT_DATE)
-    assert result == string_value
+@pytest.mark.parametrize(
+    "content_type, expected_value",
+    [(proto().CT_STRING, string_value),
+     (proto().CT_DATE, string_value),
+     (proto().CT_INT, int_value)])
+def test_protobuf_values_based_on_content_type(content_type, expected_value):
+    result = proto().value_based_on_content_type(byte_value, content_type)
+    assert result == expected_value
 
-    result = proto.value_based_on_content_type(byte_value, proto.CT_JPEG)
-    assert result == byte_value
-
-    result = proto.value_based_on_content_type(byte_value, proto.CT_PNG)
-    assert result == byte_value
-
-    result = proto.value_based_on_content_type(byte_value, proto.CT_INT)
-    assert result == int_value
 
+def test_warning_protobuf_values_based_on_content_type(proto):
```

Comment:
Should this be something like `test_protobuf_values_based_on_unknown_content_type`?